lang/ast/src/traits/occurs.rs

```diff
@@ -32,6 +32,22 @@ pub trait Occurs {
             _ => false,
         })
     }
+    /// Whether a metavariable with the given `meta_var_id` occurs as a subexpression
+    ///
+    /// # Parameters
+    ///
+    /// - `ctx`: current context under which `self` is closed
+    /// - `meta_var_id`: the metavariable id we are looking for
+    ///
+    /// # Returns
+    ///
+    /// Whether a hole with `meta_var_id` occurs as a subexpression
+    fn occurs_metavar(&self, ctx: &mut LevelCtx, meta_var_id: u64) -> bool {
```

Comment:
I wondered why we are passing the bare id instead of metavar, because I thought that metavar is just a wrapper around u64. But then I looked and discovered that we attach source spans to meta variables... 
Nothing definite to propose here, but maybe we should revisit this type definition.